src/components/CardHeader/CardHeader.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import './CardHeader.css';
+import './CardHeader.css'
 
-const CardHeader = () => {
-  return (
-    <span className='card-header'>
+const CardHeader = ()=>{
+  return(
+    <div>
```

Comment:
This change breaks tests because the className is required for the tests. Also, the removal of the semi-colons kicks back ESLint errors.